build/props/common.props

```diff
@@ -12,4 +12,7 @@
         <Version>1.0.0-preview4$(VersionSuffix)</Version>
         <Description>Silk.NET is a high-speed, advanced library, providing bindings to popular low-level APIs such as OpenGL, OpenAL, and Vulkan.</Description>
     </PropertyGroup>
+    <ItemGroup>
+        <PackageReference Include="Ultz.SuperInvoke" Version="1.0.0-preview20200202.3" /> <!-- TODO remove this! -->
```

Comment:
Is this todo meant to be here?

Reply:
Yes because at the moment it's referencing the experimental feed as there isn't a public build of SuperInvoke yet. SuperInvoke will be pushed to the public feed just before Preview 4 releases, and this will be removed from common.props and into the appropriate csprojs. This is just so I could quickly test it.

TL;DR will be rectified in a future PR

Reply:
Gotcha